### PR TITLE
Add information about matrix-params in the artifacts.properties reference

### PR DESCRIPTION
**Pull request by jgsogo**
Matrix-params for Conan are not available yet (as of Artifactory 6.16). We might wait to merge this piece of docs until the feature is released.

Docs for https://github.com/conan-io/conan/pull/6014

### CODE REVIEW

File: reference/config_files/artifacts.properties.rst [hunk 1]
```diff
@@ -3,17 +3,24 @@
...
 
    custom_header1=Value1
    custom_header2=45
+   build.name=BuildJob
+
 
-Artifactory users can use this file to set file properties for the uploaded files. The variables should have the prefix
-``artifact_property``. You can use ``;`` to set multiple values to a property:
+Artifactory users can benefit from this capability to set file properties for the uploaded files.
+If the Artifactory version doesn't support matrix params yet (available since 6.XX) it will use
```
Comment (jgsogo) [inline reference/config_files/artifacts.properties.rst]: I don't know the Artifactory release for this feature 😓

Comment (lasote) [reply reference/config_files/artifacts.properties.rst]: Yes. we have to wait.

Comment (jgsogo) [reply reference/config_files/artifacts.properties.rst]: Add a "request changes" review, so we don't merge this one accidentally

**Comment by jgsogo** [comment]
It has landed into Artifactory 7.3.2: https://www.jfrog.com/confluence/display/JFROG/Artifactory+Release+Notes#ArtifactoryReleaseNotes-Artifactory7.3.2

I'll move this PR to `master`

File: reference/config_files/artifacts.properties.rst [hunk 2]
```diff
@@ -3,17 +3,24 @@
...
    custom_header1=Value1
    custom_header2=45
+   build.name=BuildJob
+
 
-Artifactory users can use this file to set file properties for the uploaded files. The variables should have the prefix
-``artifact_property``. You can use ``;`` to set multiple values to a property:
+Artifactory users can benefit from this capability to set file properties for the uploaded files.
+If the Artifactory version doesn't support matrix params yet (available since `Artifactory 7.3.2`_) it will use
+the properties from the file that are prefixed with ``artifact_prooperty_``:
```
Comment (czoido) [inline reference/config_files/artifacts.properties.rst]:
```suggestion
the properties from the file that are prefixed with ``artifact_property_``:
```

**Pull request merged**